ooniprobe/View/RunTest/TestRunningViewController.m

```diff
@@ -169,4 +168,8 @@ -(IBAction)cancelTest:(id)sender{
                             inView:self];
 }
 
+- (void)interruptTestUI {
+    [self.runningTestsLabel setText:NSLocalizedString(@"Dashboard.Running.Stopping.Title", nil)];
+    [self.logLabel setText:NSLocalizedString(@"Dashboard.Running.Stopping.Notice", nil)];
```

Comment:
shouldn't you remove the observer here?

Reply:
In theory I should remove the observer when the View  is dismissed. Good catch

Reply:
Bot on the other hand, this is automatically done by the OS when it's destroyed

Reply:
Okay, thanks for clarifying!